### DISCUSSION TITLE
Возможность обновления карты

### DISCUSSION THREAD

**Discussion by KukushkaSex** [Sokol-Games-GmbH/PixelCombats.PublicApi#85] [Ideas]
Добавить возможность обновить карту. Тоесть, обновить саму карту (изменить), обновить имя, и тд

**Comment by forwork10** [comment]
Глупо, такое не добавим, обновлять можно только режимы будет, карты - какие выложили, такие они и будут, чтобы не получилось так, что карта стала хуже, а оценки там хорошие.

**Comment by KukushkaSex** [comment]
Спт хуй соси

**Comment by forwork10** [comment]
Report

**Comment by KukushkaSex** [comment]
А допустим забыл убрать что то в карте, или баг, а оценок много

**Comment by forwork10** [comment]
а надо было раньше думать

**Comment by justqstn** [comment]
он по факту все сказал

**Comment by KukushkaSex** [comment]
Тот же Макс бесился от этого

**Comment by forwork10** [comment]
ну тебе всеравно это не надо, ты карты не строишь

**Comment by kkohno** [comment]
тут специально сделано, чтобы нельзя было изменить карту. 
Игровые режимы, списки карт - все это зависит от конкретных карт...
Тоесть определенному ID карты должен однозначно соответствовать проверенная разработчиком режима, годная для режимв карта, чтобы не попортить игровой процесс, если вдруг владелец там что-то поменял.

У нас есть идея чтобы при выкладывании карты хранилась история версий карты с дополнительными ID.
В итоге когда вносите изменения, то старая версия никуда не девается, а просто появляется новая.
В списке карт хранить уже не только ID карты. но и ID ее версии.
Также в списках чтобы отмечалось, что такаято карта устарела на столькото версий и тамже была кнопка - поменять ссылку на новую версию. 
Также требует разработки отката версии карты в списке.
Примерно такой фронт работ, чтобы и обновления на карту выкладывать можно было и в режим карты поставлялись только проверенные разработчиками режимов.
